### PR TITLE
MOTECH:2504- Improve logging of HTTP-Agent module

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: http-agent/src/main/java/org/motechproject/http/agent/listener/HttpClientEventListener.java [hunk 1]
```diff
@@ -232,7 +232,8 @@ public void handle(MotechEvent motechEvent) {
     private ResponseEntity<?> doExecuteForReturnType(String url, Object requestData, Method method,
                                                      RestTemplate restTemplate) {
         ResponseEntity<?> response = method.execute(restTemplate, url, requestData);
-        sendAuditLog(url, requestData, response.getBody().toString(), response.getStatusCode().toString());
+        String body = (response.getBody() == null) ? "" : response.getBody().toString();
```
Comment (sebbrudzinski) [inline http-agent/src/main/java/org/motechproject/http/agent/listener/HttpClientEventListener.java]: Unnecessary parentheses

Comment (sebbrudzinski) [inline http-agent/src/main/java/org/motechproject/http/agent/listener/HttpClientEventListener.java]: We may use StringUtils.EMPTY, instead of ""

**Comment by sebbrudzinski** [comment]
test this please

**Comment by sebbrudzinski** [comment]
test this please

**Comment by sebbrudzinski** [comment]
@pgesek LGTM

**Pull request merged**